e2e/suites/viewer/viewer-actions.test.ts

```diff
@@ -74,6 +74,7 @@ describe('Viewer actions', () => {
     const fileForEditOffline = `file1-${Utils.random()}.docx`; let fileForEditOfflineId;
     const fileForCancelEditing = `file2-${Utils.random()}.docx`; let fileForCancelEditingId;
     const fileForUploadNewVersion = `file3-${Utils.random()}.docx`; let fileForUploadNewVersionId;
+    const fileForUploadNewVersion2 = `file4-${Utils.random()}.docx`; let fileForUploadNewVersionId2;
```

Comment:
what is the purpose of 'fileForUploadNewVersion2' since there is a file 'fileForUploadNewVersion' ? can we not reuse it?